lib/vendor/konami.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Konami-JS ~
```

Comment:
If we do that, we should copy the attribution parts of this comment into `easterEgg.js` (the copyright notice is required by the MIT license, the rest to be nice).